pack/gambit_encounter.json

```diff
@@ -130,6 +130,7 @@
    {
       "attack": 1,
       "code": "37034",
+      "erratum": "Removed “and put it back into play”. (RRG 1.6)",
```

Comment:
Not sure if we need smart quotes in all of these or if normal quotes will suffice. With the flavor text, we usually use normal quotes. If I put this line into a file and run the `add_octgnid.py` script to format it, it'll replace all these smart quotes with \u201c and \u201d.